### ISSUE TITLE
Error with image size

### ISSUE THREAD

**Issue by bruno-krinski** [GeorgeSeif/Semantic-Segmentation-Suite#59]
Hi, I had this error: InvalidArgumentError (see above for traceback): logits and labels must be same size: logits_size=[36864,2] labels_size=[40000,2]. I think this is why my masks are in gray scale (I have only two classes). Any suggestions that I can run without this error?

**Comment by sanja7s** [comment]
I also have similar issues with several models (but not with DeepLab-V3, for instance). 

Running **DeepLab-V3** with 1000x1000 images (grayscale or 3 channel, RGB) works fine.

With the same data and using **GCN** I get error:

InvalidArgumentError (see above for traceback): Incompatible shapes: [1,21,63,63] vs. [1,21,64,64]

and using **RefineNet-Res152** the error is:

InvalidArgumentError (see above for traceback): Incompatible shapes: [1,64,64,256] vs. [1,63,63,256]

Could you point us please if we should shape our data differently and how to adapt to different models?

Thanks!

**Comment by GeorgeSeif** [comment]
Please see issue #27
